src/lib/timeseries.spec.js

```diff
@@ -220,7 +225,7 @@ describe('Aggregation', () => {
       expect(aggregatedTimeseries.length).toBe(2)
     })
 
-    it('should return timeseries ids and totalCO2 sorted by modes', () => {
+    it('should return timeseries and totalCO2 sorted by modes witout duplicate', () => {
```

Comment:
```suggestion
    it('should return timeseries and totalCO2 sorted by modes without duplicate', () => {
```

Reply:
in my opinion, we can rename this test, "it should remove duplicates" ^^

Reply:
C'est plus que juste ne pas avoir de duplication, j'aurais pu faire un autre test spécifique mais je ne pense pas que ça soit très pertinent 🤔 
C'est surtout pour éviter de reproduire le bug corrigé :)